src/Utilities/constants.js

```diff
@@ -11,7 +11,12 @@ export const staleness = [
     { label: 'Stale', value: 'stale' },
     { label: 'Stale warning', value: 'stale_warning' }
 ];
-export const registered = [{ label: 'Insights', value: 'insights' }, { label: 'Insights not connected', value: 'nil' }];
+export const registered = [
+    { label: 'insights-client', value: 'puptoo' },
+    { label: 'subscription-manager', value: 'rhsm-conduit' },
+    { label: 'Satellite/Discovery', value: 'yupana' },
```

Comment:
Is "Discovery" a data collection agent that reports through yupana (in addition to satellite), or is that one thing?

Reply:
Yes, they both go through yupana (and I think they're separate services). The values were decided on in [ESSNTL-2443](https://issues.redhat.com/browse/ESSNTL-2443)